CMakeLists.txt

```diff
@@ -76,9 +79,12 @@ find_python_module(jinja2 REQUIRED)
 
 find_package(OpenSCAP REQUIRED)
 
-if (SSG_OVAL_511_ENABLED AND NOT "${OSCAP_V_OUTPUT}" MATCHES "OVAL Version: 5.11")
-    set(SSG_OVAL_511_ENABLED OFF CACHE BOOL "OVAL 5.11 disabled because your version of OpenSCAP doesn't support it" FORCE)
-    message(WARNING "Your version of OpenSCAP does not support OVAL 5.11, disabling OVAL 5.11 for the SSG build.")
+if (SSG_TARGET_OVAL_MAJOR_VERSION EQUAL "5" AND SSG_TARGET_OVAL_VERSION_MINOR EQUAL "11" AND NOT "${OSCAP_V_OUTPUT}" MATCHES "OVAL Version: 5.11")
+    message(FATAL_ERROR "Your version of OpenSCAP does not support OVAL 5.11, please switch the OVAL target version to 5.10 or lower. $ cmake -DSSG_TARGET_OVAL_MINOR_VERSION=10 ../")
+endif()
+
+if (NOT SSG_TARGET_OVAL_VERSION VERSION_EQUAL "5.11" AND NOT SSG_TARGET_OVAL_VERSION VERSION_EQUAL "5.11")
+    message(WARNING "You are targetting OVAL version ${TARGET_OVAL_VERSION}. In SSG we support/test 5.11 and 5.10 only!")
```

Comment:
It is comparing against "5.11" twice, and  `${TARGET_OVAL_VERSION}` should be `${SSG_TARGET_OVAL_VERSION}`

Reply:
Good catch, I was inattentive here... Fixed